cmd/commands/git-source.go

```diff
@@ -444,13 +447,16 @@ func NewGitSourceListCommand() *cobra.Command {
 				return fmt.Errorf("must enter runtime name")
 			}
 
-			return RunGitSourceList(cmd.Context(), args[0])
+			return RunGitSourceList(cmd.Context(), args[0], cmd)
 		},
 	}
+
+	cmd.Flags().BoolVar(&includeInternal, "include-internal", false, "If true, will include the Codefresh-internal git-sources")
+
 	return cmd
 }
 
-func RunGitSourceList(ctx context.Context, runtimeName string) error {
+func RunGitSourceList(ctx context.Context, runtimeName string, cmd *cobra.Command) error {
```

Comment:
why not just pass in the `bool`? I am super confused by this :/

Reply:
I can pass the bool instead, and do this operation one level sooner, np